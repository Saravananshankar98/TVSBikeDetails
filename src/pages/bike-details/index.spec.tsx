import { render } from "@testing-library/react";
import BikeDetailsPage from ".";
import { TestContextProvider } from "../../test/test-container";

describe("DashBoard", () => {
  let baseElement: any, container, screen;
  beforeEach(() => {
    ({ container, baseElement, ...screen } = render(<BikeDetailsPage />, {
      wrapper: TestContextProvider,
    }));
  });

  it("Matches Snapshot", () => {
    expect(baseElement).toMatchSnapshot();
  });
});
