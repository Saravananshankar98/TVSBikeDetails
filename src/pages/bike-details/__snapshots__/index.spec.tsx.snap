// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashBoard Matches Snapshot 1`] = `
<body>
  <div>
    <div
      class="MuiBox-root css-0"
    >
      <h2>
        CONCEPT
      </h2>
      <div
        class="MuiGrid-root MuiGrid-container css-1jzi3sl-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-grid-xs-7 css-1fdlgx2-MuiGrid-root"
        >
          <h6
            class="MuiTypography-root MuiTypography-subtitle1 css-10wpov9-MuiTypography-root"
          >
            <h2
              class="MuiTypography-root MuiTypography-h2 css-1sra7t5-MuiTypography-root"
            >
              Apache RTR 180
            </h2>
            <br />
            For over ten years the TVS Apache RTR 180 has been a mainstay on the track, pushing the boundaries of performance. Now the iconic race machine is set to become the all-new racer's choice. Itâ€™s the first in its class to be equipped with ride modes, making the already formidable performance machine, even more cutting edge. The iconic styling now gets more aggressive, with distinct LED headlamps and tail lamps. While the flared tank design and angular cowls, offer aerodynamic superiority and track presence.
          </h6>
        </div>
        <div
          class="MuiGrid-root MuiGrid-grid-xs-5 css-jwoiin-MuiGrid-root"
        >
          <img
            alt="all"
            class="MuiBox-root css-fc8lo2"
            src="https://www.tvsmotor.com/tvs-apache/-/media/Brand-Pages/Apache/Apache-RTR/Apache-RTR-180/08-09-22/optimized-images/TVS-Apache-180-Web-Banner_643-x-543-static-profile-shot.png"
          />
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root css-0"
    >
      <h2>
        Features
      </h2>
      <div
        class="MuiGrid-root MuiGrid-container css-1encfgq-MuiGrid-root"
      >
        <div
          class="MuiBox-root css-0"
        >
          <div
            class="MuiTabs-root css-1ujnqem-MuiTabs-root"
          >
            <div
              class="MuiTabs-scroller MuiTabs-fixed css-jpln7h-MuiTabs-scroller"
              style="overflow: hidden; margin-bottom: 0px;"
            >
              <div
                class="MuiTabs-flexContainer css-heg063-MuiTabs-flexContainer"
                role="tablist"
              >
                <button
                  aria-selected="false"
                  class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary css-1h9z7r5-MuiButtonBase-root-MuiTab-root"
                  role="tab"
                  tabindex="-1"
                  type="button"
                >
                  Performance
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
                <button
                  aria-selected="false"
                  class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary css-1h9z7r5-MuiButtonBase-root-MuiTab-root"
                  role="tab"
                  tabindex="-1"
                  type="button"
                >
                  Style
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
                <button
                  aria-selected="false"
                  class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary css-1h9z7r5-MuiButtonBase-root-MuiTab-root"
                  role="tab"
                  tabindex="-1"
                  type="button"
                >
                  Design
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
              </div>
              <span
                class="MuiTabs-indicator css-1aquho2-MuiTabs-indicator"
                style="left: 0px; width: 0px;"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-grid-xs-5 css-jwoiin-MuiGrid-root"
        >
          <img
            alt="all"
            class="MuiBox-root css-fc8lo2"
            src="https://www.tvsmotor.com/tvs-apache/-/media/Brand-Pages/Apache/Apache-RTR/Apache-RTR-180/08-09-22/optimized-images/TVS-Apache-180-Web-Banner_643-x-543-static-profile-shot.png"
          />
        </div>
        <div
          class="MuiGrid-root MuiGrid-grid-xs-7 css-177q6p8-MuiGrid-root"
        />
      </div>
    </div>
    <div
      class="MuiBox-root css-0"
    >
      <h2>
        COLOURS
      </h2>
      <div
        class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-grid-xs-5 css-1foajmc-MuiGrid-root"
        />
        <div
          class="MuiGrid-root MuiGrid-grid-xs-7 css-svt2eh-MuiGrid-root"
        >
          <img
            alt="all"
            class="MuiBox-root css-fc8lo2"
            src="https://www.tvsmotor.com/tvs-apache/-/media/Brand-Pages/Apache/Apache-RTR/Apache-RTR-180/08-09-22/optimized-images/TVS-Apache-180-Web-Banner_643-x-543-static-profile-shot.png"
          />
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root css-0"
    >
      <h2>
        tech specs
      </h2>
      <div
        class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-grid-xs-5 css-1foajmc-MuiGrid-root"
        />
        <div
          class="MuiGrid-root MuiGrid-grid-xs-7 css-svt2eh-MuiGrid-root"
        >
          <img
            alt="all"
            class="MuiBox-root css-fc8lo2"
            src="https://www.tvsmotor.com/tvs-apache/-/media/Brand-Pages/Apache/Apache-RTR/Apache-RTR-180/08-09-22/optimized-images/TVS-Apache-180-Web-Banner_643-x-543-static-profile-shot.png"
          />
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root css-0"
    >
      <h2>
        Price
      </h2>
      <div
        class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-grid-xs-5 css-1foajmc-MuiGrid-root"
        />
        <div
          class="MuiGrid-root MuiGrid-grid-xs-7 css-svt2eh-MuiGrid-root"
        >
          <img
            alt="all"
            class="MuiBox-root css-fc8lo2"
            src="https://www.tvsmotor.com/tvs-apache/-/media/Brand-Pages/Apache/Apache-RTR/Apache-RTR-180/08-09-22/optimized-images/TVS-Apache-180-Web-Banner_643-x-543-static-profile-shot.png"
          />
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root css-0"
    >
      <h2>
        GALLERY
      </h2>
      <div
        class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-grid-xs-4 css-3xa0dg-MuiGrid-root"
        >
          <img
            alt="all"
            class="MuiBox-root css-fc8lo2"
            src="https://www.tvsmotor.com/tvs-apache/-/media/Brand-Pages/Apache/Apache-RTR/Apache-RTR-180/08-09-22/optimized-images/TVS-Apache-180-Web-Banner_643-x-543-static-profile-shot.png"
          />
        </div>
        <div
          class="MuiGrid-root MuiGrid-grid-xs-4 css-lpzb36-MuiGrid-root"
        >
          <img
            alt="all"
            class="MuiBox-root css-fc8lo2"
            src="https://www.tvsmotor.com/tvs-apache/-/media/Brand-Pages/Apache/Apache-RTR/Apache-RTR-180/08-09-22/optimized-images/TVS-Apache-180-Web-Banner_643-x-543-static-profile-shot.png"
          />
        </div>
        <div
          class="MuiGrid-root MuiGrid-grid-xs-4 css-lpzb36-MuiGrid-root"
        >
          <img
            alt="all"
            class="MuiBox-root css-fc8lo2"
            src="https://www.tvsmotor.com/tvs-apache/-/media/Brand-Pages/Apache/Apache-RTR/Apache-RTR-180/08-09-22/optimized-images/TVS-Apache-180-Web-Banner_643-x-543-static-profile-shot.png"
          />
        </div>
      </div>
    </div>
  </div>
</body>
`;
